ts/index.ts

```diff
@@ -1,27 +1,40 @@
-// @ts-ignore
 import { NativeModules } from 'react-native'
 
-const { RNJolocom } = NativeModules;
+const { RNJolocom } = NativeModules
 
-export default RNJolocom;
-export * from './transports'
+export default RNJolocom
 export * from './storage'
 export * from './passwordStore'
 export * from './secureStorage'
 export * from '@jolocom/sdk'
 export * from 'jolocom-lib'
+
+/**
+ * Re-exporting available plugins
+ */
+export { DeepLinkingProvider } from './plugins/deep-linking/DeepLinkingProvider'
```

Comment:
To make exporting easier, we can have an `index.ts` inside `/plugins` which aggregates and exports all the plugins. As a result, here we can just do `export * from ./plugins`.

Reply:
I won't to avoid using *, it become really un-readable was is being reexported